chef/data_bags/crowbar/bc-template-provisioner.json

```diff
@@ -79,6 +79,14 @@
       "windows": {
         "admin_password": "crowbar"
       },
+      "suse": {
+        "autoyast": {
+          "packages": {
+            "suse-11_3": [ ],
+            "suse-12_0": [ ]
```

Comment:
As @jsuchome mentioned earlier:
> As with naming, it would be better consistent with values of [:autoyast][:repos] which are actually suse-11.3, suse-12.0 and common (common maybe doesn't make sense here)

(agreed we don't need common).

So better to use the dot instead of underscore. And yes, it will be an issue for what @tboerger mentioned, but I don't think we'll show that in the UI.